File: 240913/중앙값 구하기/find-the-median.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split(" ");

const a = Number(input[0]);
const b = Number(input[1]);
const c = Number(input[2]);

if ((a > b && a < c) || (a > c && a < b)) {
    console.log(a);
} else if ((b > a && b < c) || (b > c && a > b)) {
    console.log(b);
} else if ((c > a && c < b) || (a > c && c > b)){
    console.log(c);
}

File: 251027/중앙값 구하기/find-the-median.py
a, b, c = map(int, input().split())

median = None

# a > b, c
if a > b and a > c:
    if b > c: # a > b > c
        median = b
    else: # a > c > b
        median = c
elif a < b and a < c: # a < b, c
    if b < c:
        median = b # a < b < c
    else:
        median = c # a < b < c
else:
    median = a

print(median)


